thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/validators/DatasetConfigValidator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.detection.validators;
+
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import org.apache.pinot.thirdeye.datalayer.dto.DatasetConfigDTO;
+
+public class DatasetConfigValidator implements ConfigValidator<DatasetConfigDTO> {
```

Comment:
Can we have more specific name for this class, something like `OnlineDatasetConfigValidator`?

Reply:
This is removed now. Please see this [response](https://github.com/apache/incubator-pinot/pull/5769#discussion_r467201152).